integration_tests/mockApis/ciagApi.ts

```diff
@@ -1,46 +1,4 @@
-import { SuperAgentRequest } from 'superagent'
 import { stubFor } from './wiremock'
-import plans from '../mockData/ciagPlanData'
-
-const getCiagPlan = (id = 'G6115VJ') => stubFor(plans[id])
-
-const stubGetCiagPlan404Error = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
```

Comment:
I've removed the stubs to get a CIAG Plan from the CIAG API. 
This leaves the stubs for creating and updating ; but these will be removed soon with @mattwills-moj-digital's fine work 👍